control/autoware_smart_mpc_trajectory_follower/CMakeLists.txt

```diff
@@ -2,10 +2,29 @@ cmake_minimum_required(VERSION 3.14)
 project(autoware_smart_mpc_trajectory_follower)
 
 find_package(autoware_cmake REQUIRED)
+find_package(python_cmake_module REQUIRED)
 autoware_package()
+ament_python_install_package(${PROJECT_NAME})
+find_package(Python3 REQUIRED COMPONENTS Interpreter Development)
+find_package(pybind11_vendor REQUIRED)
+find_package(pybind11 REQUIRED)
+
+execute_process(COMMAND bash -c "pip3 install numba==0.58.1 --force-reinstall")
+execute_process(COMMAND bash -c "pip3 install GPy")
+
+set(PYTHON_INSTALL_DIR "./scripts")
```

Comment:
is this neccesary?

please refer https://github.com/tier4/driving_log_replayer/blob/develop/driving_log_replayer/CMakeLists.txt#L9 
CMakeLists.txt and directory structure.

Reply:
in driving_log_replayer

Reply:
Please let me confirm what you are saying.

Are you saying that we should have a `requirements.txt` file and change it to list the dependencies there?

Reply:
Oh, sorry, my link pointing to L9 is incorrect.
I was wondering why the driving log replayer does not have `set(PYTHON_INSTALL_DIR “. /scripts")` is missing and this PR needs it.